recipes/treelite/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "treelite" %}
+{% set version = "0.92" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dmlc/treelite/archive/{{ version }}.tar.gz
+  sha256: 17961a4f4c5933630dfee5c7129180e9d69660f3efa05f87bc8b168081bf3fd7
+
+build:
+  number: 0
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - cmake >=3.14
+    - ninja
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - libprotobuf >=3.10
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+  host:
+    - python
+    - pip >=19.0
+    - numpy
+    - scipy
+  run:
+    - python
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+    - {{ pin_compatible('numpy') }}
```

Are we linking to `numpy` (like using headers from `numpy`)? If so, this is fine. If not, we can just list `numpy` in `run` and drop it from `host`.

---

No, NumPy is only a run-time requirement. The C++ code does not use any header from NumPy.

---

Ah ok. Thanks for clarifying.

We can also drop `pin_compatible` as that only makes sense when we are linking against a library and want to constrain the version of the package used at runtime respectively (to ensure ABI compatibility).

```suggestion
    - numpy
```